src/picterra/__main__.py

```diff
@@ -207,6 +207,15 @@ def create_remote_raster_type(coords_string):
     delete_detectionarea_parser.add_argument(
         "raster", help="ID of the raster whose detection areas will be deleted", type=str)
 
+    # create the parsers for the "edit" command
+    edit_parser = subparsers.add_parser('edit', help="Edit resources")
+    edit_subparsers = edit_parser.add_subparsers(dest='edit')
+    # edit raster
+    edit_raster_parser = edit_subparsers.add_parser('raster', help="Edits a raster")
+    edit_raster_parser.add_argument("raster", help="ID of the raster to edit", type=str)
+    edit_raster_parser.add_argument(
+        "--name", help="Name to give to the raster", type=str, required=False)
```

Comment:
Is this not missing the other editable fields? `folder_id, captured_at, identity_key, multispectral_band_specification`?

Reply:
yep didn't want to tackle all them here, given this "picterra cli" is just for us; the main reason being that I don't want to have to escape json in the cli, so I've to figure out something like `python -m picterra edit raster <uuid> --vizbands 3 4 5 -- ranges [2, 4] [56, 67]` , but will require a lot of argparse logic so I'd postpone it